autoscheduler/frontend/src/components/LandingPage/SelectTerm.tsx

```diff
@@ -72,8 +74,8 @@ const SelectTerm: React.SFC = function App() {
           {options.map((option) => (
             <MenuItem
               key={option}
-              selected={option === 'Pyxis'}
-              onClick={handleClose}
+              selected={option === selectedTerm}
```

Comment:
Pyxis was just a dummy value. By hooking it up to state, I've made it so that the option that matched the selected term will be highlighted